### PR TITLE
Created Full Website

### PR DESCRIPTION
**Pull request by demesameneshoa**
## In This PR I Executed 🔢 
- [x] Installing React Route
- [x] Adding Routes for Each Link
- [x] Adding A Navigation Bar
- [x] Adding A Style as Per the Wireframe

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -52,11 +52,13 @@
 
 - **React App Created**
 - **Rect App Running**
```
Comment (DammyShittu) [inline README.md:54]: - [ ] Kindly edit the MIT file to be more descriptive of the owner (you) and the app type.

  image

**Review by DammyShittu — CHANGES_REQUESTED** [review]
Hi @demesameneshoa,

## Highlights
✔️ Routing feature

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (demesameneshoa) [reply README.md:54]: Done

**Review by joskalenda — APPROVED** [review]
| STATUS: APPROVED ✅ |
|------------------------|
| Hi @demesameneshoa , 🥰|
| Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: Congratulations! 🎉  |
|  |

### Highlights
✅ ... Readme file
✅ ... Great design

---------
### Optional suggestions

> _Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

### Cheers and Happy coding!👏👏👏
-------
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me(@joskalenda) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**